README.md

```diff
@@ -58,7 +60,7 @@ Then in a recipe use the `user_manage` resource to add all users in the defined
 users_manage 'GROUPNAME' do
   group_id GROUPID
   action [:create]
-  data_bag 'DATABAG_NAME'
+  users users_variable
```

Comment:
Ideally, it'd be nice to still show examples for both with and without databags to help users.

Reply:
I am showing three examples. One where I use a data bag, one where I use attributes and one where I use a local variable. So I think it is covered. If not, can you let me know what exampel you want specifically?

Reply:
Ah yes, I should have looked at the whole file as rendered. Something I think would be helpful here is to show what `users_variable` should look like or just show a simple example without using a variable.